devices/green_power.js

```diff
@@ -28,6 +25,5 @@ module.exports = [
         fromZigbee: [fz.greenpower_7],
         toZigbee: [],
         exposes: [e.action(['*'])],
-        whiteLabel: [{vendor: 'EnOcean', description: 'Easyfit 1 or 2 gang switch', model: 'EWSxZG'}],
```

Comment:
Just did some test with my Hue tap and PTM215Z and it works perfectly, I'm really happy with this solution.

Is there any reason to keep `green_power.js` and its converters? The only downside I see here that it might make an existing supported device unsupported, but I think chances are small. So if you agree please remove it.